storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Components/VerificationResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Dotnet.Storm.Adapter.Components
+{
+    public class VerificationResult
+    {
+        private string reason;
+
+        internal VerificationResult(bool error, string message)
+        {
+            IsError = error;
+            reason = message;
+        }
+
+        public bool IsError { get; private set; }
+
+        public string Reason {
+            get
+            {
+                return ToString();
+            }
+            private set
+            {
+                reason = value;
+            }
+        }
+
+        public override string ToString()
+        {
+            string prefix = IsError ? "Verification failed." : "Verification passed.";
+            return string.Format("{0} {1}", prefix, reason);
```

Comment:
More compact syntax
return $"{prefix} {reason}";

Reply:
actually it could be simplified. See my next commit.